sensor_msgs/Image image
geometry_msgs/Pose cartesian
bool finded